$NetBSD: patch-ac,v 1.1 2008/11/08 13:47:11 bjs Exp $

Add these definitions here to avoid namespace pollution in
include/mutils/mhash_config.h.

--- include/mutils/config.h.in.orig	2008-11-08 08:23:23.000000000 -0500
+++ include/mutils/config.h.in
@@ -0,0 +1,22 @@
+/* Name of package */
+#undef PACKAGE
+
+/* Define to the address where bug reports for this package should be sent. */
+#undef PACKAGE_BUGREPORT
+
+/* Define to the full name of this package. */
+#undef PACKAGE_NAME
+
+/* Define to the full name and version of this package. */
+#undef PACKAGE_STRING
+
+/* Define to the one symbol short name of this package. */
+#undef PACKAGE_TARNAME
+
+/* Define to the version of this package. */
+#undef PACKAGE_VERSION
+
+/* Version number of package */
+#undef VERSION
+
+
